llvm/include/llvm/Object/ObjectFile.h

```diff
@@ -302,6 +302,7 @@ class ObjectFile : public SymbolicFile {
 public:
   ObjectFile() = delete;
   ObjectFile(const ObjectFile &other) = delete;
+  ObjectFile operator=(const ObjectFile &other) = delete;
```

Comment:
`ObjectFile &opeartor=`

Reply:
This should probably be `&operator` and not `&opeartor`. I think this typo is causing nearly every buildbot to fail.